kv/etcd/store.go

```diff
@@ -163,15 +163,16 @@ func (c *client) get(key string) (kv.Value, error) {
 }
 
 func (c *client) History(key string, from, to int) ([]kv.Value, error) {
-	if from >= to || from < 0 || to < 0 {
+	if from > to || from < 0 || to < 0 {
 		return nil, errInvalidHistoryVersion
 	}
 
```

Comment:
As a simple optimization, add `if from == to { return nil, nil }` to avoid unnecessary work